.github/.wordlist.txt

```diff
@@ -886,6 +890,7 @@ npm
 nRF
 nrfconnect
 nrfdks
+nrfjprog
```

Comment:
nrfjprog is not a word

Reply:
`nrfjprog` is a Nordic tool for programming through SEGGER J-LINK programmers and debuggers, but I guess I can delete it from a wordlist and add backticks.